commune_app/models.py

```diff
@@ -1,3 +1,12 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
```

Comment:
please it will be better to create the user based on:
from django.contrib.auth.models import AbstractUser
```suggestion
class User(AbstractUser):
```
in this way, you will get many functionalities for free